module/input/NatNet/src/NatNet.cpp

```diff
@@ -307,37 +329,55 @@ namespace module::input {
         emit(std::move(mocap));
     }
 
-    void NatNet::processModel(const Packet& packet) {
+    void NatNet::process_model(const Packet& packet) {
 
         log<NUClear::INFO>("Updating model definitions");
 
         // Our pointer as we move through the data
         const char* ptr = &packet.data;
 
-        uint32_t nModels = ReadData<uint32_t>::read(ptr, version);
+        uint32_t n_models = ReadData<uint32_t>::read(ptr, version);
 
-        for (uint32_t i = 0; i < nModels; ++i) {
+        for (uint32_t i = 0; i < n_models; ++i) {
             // Read the type
             uint32_t type = ReadData<uint32_t>::read(ptr, version);
 
             // Parse the correct type
             switch (type) {
                 // Marker Set
                 case 0: {
-                    MarkerSetModel m        = ReadData<MarkerSetModel>::read(ptr, version);
-                    markerSetModels[m.name] = m;
+                    MarkerSetModel m          = ReadData<MarkerSetModel>::read(ptr, version);
+                    marker_set_models[m.name] = m;
                 } break;
 
                 // Rigid Body
                 case 1: {
-                    RigidBodyModel m      = ReadData<RigidBodyModel>::read(ptr, version);
-                    rigidBodyModels[m.id] = m;
+                    RigidBodyModel m        = ReadData<RigidBodyModel>::read(ptr, version);
+                    rigid_body_models[m.id] = m;
                 } break;
 
                 // Skeleton
                 case 2: {
-                    SkeletonModel m      = ReadData<SkeletonModel>::read(ptr, version);
-                    skeletonModels[m.id] = m;
+                    SkeletonModel m       = ReadData<SkeletonModel>::read(ptr, version);
+                    skeleton_models[m.id] = m;
+                } break;
+
+                // Force Plate
+                case 3: {
+                    ForcePlateModel m        = ReadData<ForcePlateModel>::read(ptr, version);
+                    force_plate_models[m.id] = m;
+                } break;
+
+                // Device
+                case 4: {
+                    DeviceModel m       = ReadData<DeviceModel>::read(ptr, version);
+                    device_models[m.id] = m;
+                } break;
+
+                // Camera
+                case 5: {
+                    CameraModel m = ReadData<CameraModel>::read(ptr, version);
+                    camera_model  = m;
```

Comment:
Is the camera model only ever going to appear once? It seems that data would be lost if not?